projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-quote.ts

```diff
@@ -1758,3 +1758,60 @@ export function reload() {
 function log(comment: string, functionName: string) {
   cy.log(`##### ${comment} <${functionName}> #####`);
 }
+
+/**
+ * Registers read vendor quote route.
+ */
+export function registerReadVendorQuoteRoute() {
+  log('Registers read vendor quote route.', registerReadVendorQuoteRoute.name);
+  cy.intercept({
+    method: 'GET',
+    path: `${Cypress.env('OCC_PREFIX')}/${SHOP_NAME}/users/*/quotes*`,
+  }).as(READ_VENDOR_QUOTE.substring(1)); // strip the '@'
+}
+
+/**
+ * Navigates to the vendor quote list via my account.
+ */
+export function navigateToVendorQuoteListFromMyAccount() {
```

Comment:
Since I suggested to use other method, please delete these two unused methods: `registerReadVendorQuoteRoute` and `navigateToVendorQuoteListFromMyAccount`